src/complex/Core/Application.cpp

```diff
@@ -141,6 +141,13 @@ std::vector<Uuid> Application::getSimplUuid(const Uuid& complexUuid)
   return uuidList;
 }
 
+inline bool ends_with(std::string const& value, std::string const& ending)
```

Comment:
```suggestion
bool ends_with(std::string_view value, std::string_view ending)
```
Maybe we should add this to the string utilities file.